### PR TITLE
Update FAQ section in documentation

### PR DESCRIPTION
**Pull request by AaronWard**
- Requested by @sonichi to add the workaround to "gratitude loops" to the FAQ
- I also removed a redundant section that focus only on `config_list_from_json`, repointing users to the `oai` utils example notebook for more clarity. 

## Why are these changes needed?

The following updates provide a workaround for  `gpt-3.5-turbo` agent models congratulating and thanking each other in a continuous loop when they complete a task. 

## Related issue number

https://discord.com/channels/1153072414184452236/1153072414184452243/1159897545946964149

## Checks

- [X] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Review by gagb — APPROVED** [review]
Thank you the changes look good.

File: website/docs/FAQ.md [hunk 1]
```diff
@@ -160,3 +87,24 @@ Otherwise, reply CONTINUE, or the reason why the task is not solved yet."""
 ```
 
 If you have problems with agents running `pip install` or get errors similar to `Error while fetching server API version: ('Connection aborted.', FileNotFoundError(2, 'No such file or directory')`, you can choose **'python:3'** as image as shown in the code example above and that should solve the problem.
+
+
+### Agents keep thanking eachother when using `gpt-3.5-turbo`
```
Comment (sonichi) [inline website/docs/FAQ.md:92]:
```suggestion
### Agents keep thanking each other when using `gpt-3.5-turbo`
```

File: website/docs/FAQ.md [hunk 2]
```diff
@@ -160,3 +87,24 @@ Otherwise, reply CONTINUE, or the reason why the task is not solved yet."""
...
+
+When using `gpt-3.5-turbo` you may often encounter agents going into a "gratitude loop", meaning when they complete a task they will begin congratulating and thanking eachother in a continuous loop. This is a limitation in the performance of `gpt-3.5-turbo`, in contrast to `gpt-4` which has no problem remembering instructions. This can hinder the experimentation experience when trying to test out your own use case with cheaper models.
+
+A workaround is to add an additional termination notice to the prompt. This acts a "little nudge" for the LLM to remember that they need to terminate the conversation when their task is complete. You can do this by appending a string such as the following to your user input string:
+
+```python
+prompt = "Some user query"
+
+termination_notice = (
+    '\n\nDo not say show appreciation in your responses, say only what is necessary. '
```
Comment (sonichi) [inline website/docs/FAQ.md:102]:
```suggestion
    '\n\nDo not say appreciation in your responses, say only what is necessary. '
```

File: website/docs/FAQ.md [hunk 3]
```diff
@@ -160,3 +87,24 @@ Otherwise, reply CONTINUE, or the reason why the task is not solved yet."""
...
+    'if "Thank you" or "You\'re welcome" are said in the conversation, then say TERMINATE '
+    'to indicate the conversation is finished and this is your last message.'
+)
+
+prompt += termination_notice
+```
+
+**Note**: This workaround gets the job done around 90% of the time, but there are occurences where the LLM's will still forget to terminate the conversation.
```
Comment (sonichi) [inline website/docs/FAQ.md:110]:
```suggestion
**Note**: This workaround gets the job done around 90% of the time, but there are occurences where the LLM still forgets to terminate the conversation.
```

File: website/docs/FAQ.md [hunk 4]
```diff
@@ -2,88 +2,15 @@
...
-- Azure OpenAI API base: os.environ["AZURE_OPENAI_API_BASE"] or `aoai_api_base_file="base_aoai.txt"`. Multiple bases can be stored, one per line.
-
-It's OK to have only the OpenAI API key, or only the Azure OpenAI API key + base.
-
-```python
-import autogen
-config_list = autogen.config_list_from_models(model_list=["gpt-4", "gpt-3.5-turbo", "gpt-3.5-turbo-16k"])
-```
-
-> For Azure the model name refers to the OpenAI Studio deployment name.
```
Comment (sonichi) [inline website/docs/FAQ.md:68]: Is this sentence or a similar instruction included in the notebook?

File: website/docs/FAQ.md [hunk 5]
```diff
@@ -2,88 +2,15 @@
...
-        'model': 'gpt-3.5-turbo',
-        'api_key': '<your OpenAI API key here>',
-    },  # OpenAI API endpoint for gpt-3.5-turbo
-    {
-        'model': 'gpt-3.5-turbo-16k',
-        'api_key': '<your OpenAI API key here>',
-    },  # OpenAI API endpoint for gpt-3.5-turbo-16k
-]
-```
+We suggest that you take a look at this [notebook](https://github.com/microsoft/autogen/blob/main/notebook/oai_openai_utils.ipynb) for full code examples of the different methods to configuring your model endpoints.
```
Comment (sonichi) [inline website/docs/FAQ.md:13]:
```suggestion
We suggest that you take a look at this [notebook](https://github.com/microsoft/autogen/blob/main/notebook/oai_openai_utils.ipynb) for full code examples of the different methods to configure your model endpoints.
```

File: website/docs/FAQ.md (hunk 4 above)
Comment (AaronWard) [reply website/docs/FAQ.md:68]: No i don't think so, will add the appropriate information if not included.

**Pull request merged**